platform/lib/routers/pages.js

```diff
@@ -208,6 +211,13 @@ pages.get('/*', async (req, res, next) => {
   // for this use case
   renderedTemplate = rewriteLinks(url.pathname, renderedTemplate, templateContext.format);
 
+  // Pipe the rendered template through the AMP optimizer
+  try {
```

Comment:
Not too sure if the optimizer's promise is ever rejected or if throws in general, you probably know better, @sebastianbenz 😉

Reply:
I could fail so it's good to cache the error and return the original page.